src/components/navbar/index.tsx

```diff
@@ -93,7 +93,13 @@ const NavBar: React.FC<NavBarProps> = ({ tokens }) => {
   // Dropdown menu options for the logged in
   const userMenu = (
     <Menu>
-      <Menu.Item>Change Primary Account Email</Menu.Item>
+      <Menu.Item
+        onClick={() => {
+          history.push(Routes.CHANGE_ACCOUNT_EMAIL);
+        }}
+      >
+        Change Primary Account Email
+      </Menu.Item>
       <Menu.Item>Account Details</Menu.Item>
       <Menu.Item>Change Password</Menu.Item>
       <Menu.Item
```

Comment:
We should probably just delete menu items that don't go anywhere

Reply:
I think the goal is to create what we have in the Vue version - just have Settings item in the navbar, and the settings page has all the buttons to the individual setting pages (change email, change password, edit family info, etc). So in short, yes these will go away